beetsplug/absubmit.py

```diff
@@ -113,12 +128,22 @@ def analyze_submit(self, item):
 
     def _get_analysis(self, item):
         mbid = item['mb_trackid']
+
         # If file has no mbid skip it.
+        if not self.opts.force_refetch and not self.config['force']:
+            mood_str = item.get('mood_acoustic', u'')
+            if mood_str:
+                return None
```

Comment:
This simpler version would also suffice:

```
if not item.get('mood_acoustic'):
    return None
```

Maybe it would also make sense to move this "probe field" name to a module-level constant?